packages/shared/src/modules3rdParty/cross-crypto/index.js

```
@@ -16,16 +16,17 @@ if (process.env.NODE_ENV !== 'production') {
 
 const crypto = require('crypto-browserify');
 
-if (global.crypto) {
-  global.crypto.randomBytes = global.crypto.randomBytes || crypto.randomBytes;
+if (globalThis.crypto) {
```

ESLint

[error] 'globalThis' is not defined.

(no-undef)